src/pages/index.tsx

```diff
@@ -206,6 +206,12 @@ const SectionBuiltOn = styled.section`
     }
 `
 
+const UnstyledButton = styled.button`
+    border: none;
+    padding: 0;
+    margin: 0;
```

Comment:
The blue outline @svenefftinge showed is actually visible in Chrome when you click on any of these items.

I don't think it looks good, so we should also "unstyle" the `outline` property:

```suggestion
    margin: 0;
    outline: 0;
```